core/templates/dev/head/components/summary_tile/CollectionSummaryTileDirective.js

```diff
@@ -46,11 +51,21 @@ oppia.directive('collectionSummaryTile', [function() {
         };
 
         $scope.getCollectionLink = function() {
-          return UrlInterpolationService.interpolateUrl(
-            COLLECTION_VIEWER_URL, {
-              collection_id: $scope.getCollectionId()
+          if (angular.isDefined($scope.inCreatorDashboard)) {
+            if ($scope.inCreatorDashboard) {
+              return UrlInterpolationService.interpolateUrl(
+                COLLECTION_EDITOR_URL, {
```

Comment:
inCreatorDashboard is a boolean variable which I use to redirect to editor page. Do I refactor it to `linkToEditorPage`?